requirements.txt

```diff
@@ -0,0 +1,6 @@
+flask
+Flask-Cors
+-e git+https://github.com/RedisGraph/redisgraph-py#egg=redisgraph
+-e git+https://github.com/HTTP-APIs/hydra-python-core#egg=hydra_python_core
+-e git+https://github.com/HTTP-APIs/hydra-python-agent@develop#egg=hydra_agent
```

Comment:
I've created a PR on the agent to sync develop with the master. @Guttz please review that, then change the branch here to master and then we can merge.

Reply:
Okay! I'll work on the documentation and double-check everything the next days so we merge both of them with proper docs.